scripts/qa_crowbarsetup.sh

```diff
@@ -2942,6 +2950,16 @@ function deploy_single_proposal()
 {
     local proposal=$1
 
+    # skip proposals if they are explicitly disabled via
+    #   export want_$proposal=0
+    if eval "[[ \$want_${proposal} = 0 ]]" ; then
+        echo "Proposal SKIP: proposal '$proposal' will not be deployed because \$want_${proposal} was set to 0"
+        return
+    else
+        # set the default to allow other code to query its value
+        eval "want_${proposal}=1"
```

Comment:
What if we need this before we attempt to deploy this proposal? (no concrete example in mind right now)